### ISSUE TITLE
Authentication Problem

### ISSUE THREAD

**Issue by rzlamrr** [robertodoering/harpy#690]
I compiled harpy ([commit](https://github.com/robertodoering/harpy/commit/e4563b6791c3efff23cec268a6fa5b08b1123650)) with my own api key. After successful login, the application returns to the home page where there is a 'login with twitter' button

I see that this [line](https://github.com/robertodoering/harpy/blob/e4563b6791c3efff23cec268a6fa5b08b1123650/lib/components/authentication/provider/login_provider.dart#L83) being called after successful login

I have tried to enable both OAuth 1.0a and OAuth 2.0

**Comment by robertodoering** [comment]
Have you set `harpy://` as the callback url in your api settings?
https://github.com/robertodoering/harpy/wiki/Twitter-key-setup

**Comment by rzlamrr** [comment]
Of course, I won't be able to login if I don't add `harpy://` as a callback:)

**Comment by robertodoering** [comment]
You need access to the v1.1 endpoints. You either need elevated access or you might be able to create a "Standalone app" in the Twitter developer portal.
https://developer.twitter.com/en/docs/projects/overview

**Comment by rzlamrr** [comment]
unfortunately, my application was rejected so I can't get elevated access or standalone app🥲

**Comment by robertodoering** [comment]
That's unfortunate :( I will update the wiki page to point out the requirements.

**Comment by yaar007** [comment]
is there any update on this? Can this be solved without api level elevated access?
